docs/source/release_notes.rst

```diff
@@ -5,6 +5,7 @@ Release Notes
 **Future Release**
     * Enhancements
     * Fixes
+        * Update ``EntitySet.add_last_time_indexes`` to work with Koalas 1.3.0 (:pr:`1202`)
```

Comment:
Can we remove the "Restrict koalas version" entry in the release notes and and add its PR  number to this entry?

Reply:
Done!